docs/ProjectPlanning.md

```diff
@@ -124,10 +144,37 @@ Least constraints here- it will serve as an API endpoint for clients and admin i
 - Classrooms
 - Teachers
 - Students
-- Subjects
-- Classes
+- Subjects (Is this relevant for attendance purposes?)
```

Comment:
Not immediately but this may help provide additional context as the system grows